armi/bookkeeping/db/database.py

```diff
@@ -390,9 +390,13 @@ def loadCS(self, handleInvalids=True):
         custom Application you do not have access to, the DB may not be usable.
         """
         cs = settings.Settings()
-        cs.caseTitle = os.path.splitext(os.path.basename(self.fileName))[0]
+        cs.path = self.fileName
```

Comment:
This is an API-breaking change, but I do not see the need for it.

People use the caseTitle, and this change would break it for most people's workflows. I think if you want to remove the Settings.caseTitle, that should be a different PR.

Reply:
Okay, that setting/getter logic is at least 6 years old. So changing that is a probably a bad solution. ok